neuralprophet/plot_forecast_matplotlib.py

```diff
@@ -470,8 +470,37 @@ def plot_nonconformity_scores(scores, alpha, q, method):
     ax.plot(confidence_levels, scores, label="score")
     ax.axvline(x=1 - alpha, color="g", linestyle="-", label=f"(1-alpha) = {1-alpha}", linewidth=1)
     ax.axhline(y=q, color="r", linestyle="-", label=f"q1 = {round(q, 2)}", linewidth=1)
+    ax.set_title(f"{method} One-Sided Interval Width with q")
     ax.set_xlabel("Confidence Level")
     ax.set_ylabel("One-Sided Interval Width")
-    ax.set_title(f"{method} One-Sided Interval Width with q")
     ax.legend()
     return fig
+
+
+def plot_interval_width_per_timestep(q_hats, method):
+    """Plot the nonconformity scores as well as the one-sided interval width (q).
+
+    Parameters
+    ----------
+        q_hats : list
+            prediction interval widths (or q) for each timestep
+        method : str
+            name of conformal prediction technique used
+
+            Options
+                * (default) ``naive``: Naive or Absolute Residual
+                * ``cqr``: Conformalized Quantile Regression
+
+    Returns
+    -------
+        matplotlib.pyplot.figure
+            Figure showing the q-values for each timestep
+    """
+    fig, ax = plt.subplots()
+    ax.plot(range(1, len(q_hats) + 1), q_hats)
```

Comment:
I am sorry, I am a bit confused about what is being plotted based on the docstring and code.
Aren't the qhats static? 
Or what are we plotting here?

Reply:
This is the `plot_interval_width_per_timestep()` method for models with n_lags > 1 and thus self.n_forecasts >1. You can try with the m3 and m4 models in the [uncertainty_conformal_prediction.ipynb](https://github.com/ourownstory/neural_prophet/blob/main/tutorials/feature-use/uncertainty_conformal_prediction.ipynb). Therefore, instead of plotting the _Naive One-Side Interval Width from q_ (with the noncon scores and q1 horizontal line from just forecast1), this method will instead plot the q values for each of the timesteps:
![Naive One-Sdied Interval Width with q](https://user-images.githubusercontent.com/13785749/212241798-0559d5d4-3082-4d97-9e31-b2207543558f.png)
You can see how q linearly increases the further the timestep (until it plateaus around t+12)? This shows the greater uncertainty, the further out the forecast that it requires wider intervals for the same confidence level (e.g., 90% in this example).

Reply:
Thank you for explaining! 
Sorry, I interpreted x-axis to be datetime, not forecast step numbers.
Now it makes sense. Cool how one can see it increase and then plateau!